sda-orch/README.md

```diff
@@ -28,13 +30,20 @@ Parameter | Description | Default
 `broker.vhost` | Virtual host to connect to. | `/`
 `broker.password` | Shared password to the message broker. | `""`
 `broker.username` | Shared user to the message broker. | `""`
-`broker.queue.inbox` | Inbox queue for MQ connection. | `""`
-`broker.queue.completed` | Completed queue for MQ connection. | `""`
-`broker.queue.verify` | Verify queue for MQ connection. | `""`
-`broker.queue.stableid` | Accession IDs queue for MQ connection. | `""`
-`broker.queue.files` | Files queue for MQ connection, general file operations. | `""`
-
-
+`broker.queue.inbox` | Inbox queue for MQ connection. | `"inbox"`
+`broker.queue.completed` | Completed queue for MQ connection. | `"completed"`
+`broker.queue.verify` | Verify queue for MQ connection. | `"verify"`
+`broker.queue.accessionids` | Accession IDs queue for MQ connection. | `"accessionIDs"`
+`broker.queue.mapping` | Mappings for Accession IDs to DatasetIDs queue for MQ connection. | `"mappings"`
+`datasetID.external` | if the datasetids will be used by external system set this to true  | false
+`datasetID.customConfig` | if custom config is enabled it expects a file in file with the name config.json so that it can load it as configuration | false
```

Comment:
Can this be rephrased so even I can understand? :)

Reply:
hope it is better now.

Reply:
Hrrm, howabout something

If customConfig is enabled files/config.json will be used to provide a ConfigMap mounted at /config

or similar?